www/front_src/src/Resources/Details/tabs/Services/List.tsx

```diff
@@ -52,6 +52,7 @@ const ServiceList = ({
             <Paper className={classes.serviceCard}>
               <div className={classes.serviceDetails}>
                 <StatusChip
+                  clickable={false}
```

Comment:
I can’t really think of a place where a StatusChip would be clickable, so we wight want to set it to false by default in @centreon-ui

Reply:
I thought it was used by BAM for enabled/disabled icon, but it's not.
So I'll move default value to false